xml/ay_12_migrate_15.xml

```diff
@@ -394,6 +394,16 @@
    </para>
   </sect2>
 
+  <sect2 xml:id="sec.ay_12vs15.partitioning.forcing-primary">
+   <title>Forcing primary partitions</title>
+   <para>
+    It is still possible to force a partition as <literal>primary</literal>
+    (only on MS-DOS partition tables) setting the <tag class="element">primary_type</tag> 
+    to <literal>primary</literal>. However, any other value, like <literal>logical</literal>,
+    will be ignored by AutoYaST, which will decide on the partition type.
```

Comment:
...will be ignored by AutoYaST, which will automatically determine the partition type.